README.md

```diff
@@ -5,15 +5,47 @@
 Hermetic is a Slack bot for YouTrack integration. It has the following features:
 
 - Reply with an issue overview attachment whenever an issue code is mentioned.
-- Slash commands:
-	- `/yt-add ProjectID [@assignee] [#tag] Title text` Add a YouTrack issue.
-	- `/yt-cmd issue-ID youtrack command` Run a YouTrack command on an issue.
+- Add a new YouTrack issue right from the chat.
+- Run an arbitrary YouTrack command on an issue right from the chat.
 
-## Contributing
-
-Copy `config/example-config.exs` to `config/config.exs` and fill in the secrets.
+## Build instructions
 
 Run `nix-shell` in the root of the repo to pull in elixir. Run `mix deps.get` to
 pull in dependencies and install hex if asked. Run `iex -S mix` to get a repl,
 `mix test` to check no tests are breaking and `mix dialyzer` to do type
 checking. The last one takes a while on the first run.
+
+## Usage
+
+### Adding Hermetic to the Slack
+
+Copy `config/example-config.exs` to `config/config.exs` and fill in the secrets
+(Slack, Youtrack and Github API tokens).
+
+[Set up a Slack app](https://api.slack.com/slack-apps) and create two slash
+commands that point to `https://yourdomain.tld/yt-add` and
+`https://yourdomain.tld/yt-cmd`. Make sure to enable user escaping so Hermetic
+can translate Slack users (by email) to YouTrack users. You can use `nginx` for
+example to handle `https` and routing.
+
+### Using the bot
+
+Run those slash commands in a chat with Hermetic:
+
+- `/yt-add ProjectID [@assignee] [#tag] Title text` Add a YouTrack issue. A link to a previous message in the chat is added to an issue description.
+- `/yt-cmd issue-ID youtrack command` Run a YouTrack command on an issue.
+
+Also, Hermetic will automatically reply to any message containing `issue-ID` (e.
+g. `AD-150`) with a link to this issue.
+
+## Issue tracker
+
+We use [YouTrack](https://issues.serokell.io/issues/OPS) as our issue tracker.
```

Comment:
I believe this youtrack project isn't publicly accessible, but correct me if I'm wrong. It may be a good idea to create a publicly accessible youtrack project for the various open source tools we maintain, if there isn't one already.

Reply:
Hermentic is handled in INT, not OPS anyway. But perhaps it's worth having a separate project? /cc @kirelagin

Reply:
We have more open source tools and libraries that could probably use a public youtrack project for issues, but one for each might be too much.

Reply:
In that case maybe a `Public` project on YouTrack might be the way to go. I'll see if we can discuss this with management.